### PR TITLE
Add Expected Value

### PR DESCRIPTION
**Pull request by HaccerKat**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [x] I have linked this PR to any issues that it closes.

Added some basic expected value content. Also modified some of the difficulty labels that were either incorrectly written or too high for the problem difficulty.

### CODE REVIEW

File: content/4_Gold/Combinatorics.mdx [hunk 1]
```diff
@@ -887,6 +887,49 @@ int main() {
 	}
 }
 ```
+## Expected Value
+
+An expected value is the theoretical mean of a probability distribution. A random variable is used to represent a possible probability distribution. Let $X$ be a random variable and $P(X = x)$ be the probability that the result of the random variable $X$ is $x$. Then, the expected value of $X$, denoted as $E[X]$, is
```
Comment (envyaims) [inline content/4_Gold/Combinatorics.mdx:892]: italize "random variable" so they know its a term

**Review by envyaims — APPROVED** [review]
looks good, would like a section dedicated to EV practice problems

**Review by dongliu0426 — APPROVED** [review]
ok

File: content/4_Gold/Combinatorics.mdx (hunk 1 above)
Comment (HaccerKat) [reply content/4_Gold/Combinatorics.mdx]: i assume you mean bold cause that is what i see everywhere else but ok

File: content/4_Gold/Combinatorics.problems.json [hunk 2]
```diff
@@ -36,7 +36,22 @@
       "name": "MARBLES - Marbles",
       "url": "https://www.spoj.com/problems/MARBLES/",
       "source": "SPOJ",
-      "difficulty": "Medium",
+      "difficulty": "Easy",
```
Comment (SansPapyrus683) [inline content/4_Gold/Combinatorics.problems.json:39]: crazy

Comment (HaccerKat) [reply content/4_Gold/Combinatorics.problems.json:39]: is it not the most standard stars and bars problem ever :skull:

Comment (HaccerKat) [reply content/4_Gold/Combinatorics.problems.json:39]: especially when the template derangements gets "easy"

Comment (SansPapyrus683) [reply content/4_Gold/Combinatorics.problems.json:39]: fine, i'll allow it

**Pull request merged**